### PR TITLE
build: remove publicPath from webpack dev config

### PR DESCRIPTION
**Pull request by johnpaulmedina**
Added rootDir const to webpack dev config. Should resolve issue with build we discussed on #2167

### CODE REVIEW

**Review by thijstriemstra — CHANGES_REQUESTED** [review]
thanks @johnpaulmedina. Previously localhost:8080 showed a file listing of the rootdir and after some tweaking this is what most resembles prev situation:

```diff
diff --git a/build-config/fragments/dev.js b/build-config/fragments/dev.js
index 57bcc0e..c432996 100644
--- a/build-config/fragments/dev.js
+++ b/build-config/fragments/dev.js
@@ -1,6 +1,7 @@
 /* eslint-env node */
 
 const path = require('path');
+const rootDir = path.resolve(__dirname, '..', '..');
 
 module.exports = {
     mode: 'development',
@@ -11,9 +12,8 @@ module.exports = {
     devServer: {
         static: [
             {
-                directory: path.resolve(__dirname, '..', '..'),
+                directory: rootDir,
                 staticOptions: {},
-                publicPath: '/dist/',
                 serveIndex: true,
                 watch: {
                     ignored: [
```

Otherwise browsing to localhost:8080 will result in a GET error. Can you make test and make these changes?

**Review by johnpaulmedina — COMMENTED** [review]
Got it. I tested it and it now serves the root dir as requested. So the localhost can access either directory now as previously.

File: build-config/fragments/dev.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
 const path = require('path');
+const rootDir = path.resolve(__dirname, '..', '..');
 
 module.exports = {
     mode: 'development',
     devtool: 'eval-source-map',
     output: {
-        publicPath: 'localhost:8080/dist/'
+        publicPath: 'localhost:8080/'
```
Comment (thijstriemstra) [inline build-config/fragments/dev.js:10]: not sure if this var is actually used?

**Review by thijstriemstra — APPROVED** [review]
thanks!

**Pull request merged**